app/lib/get-workflows-in-order.js

```diff
@@ -0,0 +1,52 @@
+function getWorkflowsInOrder(project, query) {
+  query = query || {};
+  const order = (project.configuration && project.configuration.workflow_order) ?
+    project.configuration.workflow_order :
+    [];
+
+  // TODO remove default page_size once pagination solution implemented
+  query.page_size = query.page_size || 20;
+
+  function getAllWorkflows(query) {
+    let allWorkflows = [];
+    return getWorkflows(query, 1);
+
+    function getWorkflows(query, page) {
+      return project.get('workflows', Object.assign({}, query, { page }))
+        .then((workflows) => {
+          allWorkflows = allWorkflows.concat(workflows);
+          const meta = workflows[0] ? workflows[0].getMeta() : null;
+          if (meta && meta.page !== meta.page_count) {
```

Comment:
I did not know that. That changes code like this in a few other places. Pinging @mcbouslog because he could use this for his organisation queries too.